### PR TITLE
[MAJOR] Clean names to match GraphQL spec

### PR DESCRIPTION
**Pull request by maxnachlinger**
Addresses https://github.com/walmartlabs/json-to-simple-graphql-schema/issues/9

### CODE REVIEW

File: lib/to-schema.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const isNumber = require("lodash.isnumber");
 const isObject = require("lodash.isobject");
 const _set = require("lodash.set");
```
Comment (maxnachlinger) [inline lib/to-schema.js:24]: `key` argument wasn't used

File: lib/to-schema.js [hunk 2]
```diff
@@ -35,21 +35,26 @@ const transformPrimitive = (key, value) => {
   return "String";
 };
```
Comment (maxnachlinger) [inline lib/to-schema.js:37]: Saving a Google, which I find myself doing for Regex :)`\W` - Matches any character that is not a word character from the basic Latin alphabet

File: test/bad-character-input.spec.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/**
```
Comment (maxnachlinger) [inline test/bad-character-input.spec.js:1]: Proves that bad characters are replaced.

File: test/complex-json.spec.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
+/**
```
Comment (maxnachlinger) [inline test/complex-json.spec.js:1]: This complex input was covered in an integration test, was simple to make a unit test using it as well.

**Review by ChrisCrewdson — APPROVED** [review]
Looks good to me.

**Pull request merged**